tests/integration/ApiClient/ApiClientTest.php

```diff
@@ -22,17 +22,23 @@ public function testApiClient()
     {
         $client = $this->getClient(self::API_CLIENTS_SCOPE);
         $project = $client->getConfig()->getProject();
+        $deleteDaysAfterCreation = 1;
 
         $apiClientDraft = ApiClientDraft::ofNameAndScope(
             'test-' . $this->getTestRun(),
             'view_products:' . $project
-        );
+        )->setDeleteDaysAfterCreation($deleteDaysAfterCreation);
+
         $request = ApiClientCreateRequest::ofDraft($apiClientDraft);
         $response = $request->executeWithClient($client);
-        $result = $request->mapResponse($response);
+        $result = $request->mapFromResponse($response);
 
         $this->assertNotNull($result);
         $this->assertNotNull($result->getId());
+
+        $calcDate = date_modify(new \DateTime(), '+' . $deleteDaysAfterCreation . 'day');
```

Comment:
no need to date_modify
```suggestion
        $calcDate = new \DateTime('+' . $deleteDaysAfterCreation . 'day');
```